src/frontend/src/lib/components/registry/players/PlayerTournamentHistory.svelte

```diff
@@ -0,0 +1,228 @@
+    }
+    if (to) {
+      filtered_team_placements = filtered_team_placements.filter((item) => {
+        return item.date_end <= Date.parse(to) / 1000;
+      });
+    }
+
+    // Sorting
+    filtered_solo_placements = filtered_solo_placements.sort((a, b) => {
+      return b.date_start - a.date_start;
+    });
+
+    filtered_team_placements = filtered_team_placements.sort((a, b) => {
+      return b.date_start - a.date_start;
+    });
+  }
+
+  onMount(fetchData);
+</script>
+
+<Section header={$LL.TOURNAMENT_HISTORY.TOURNAMENT_HISTORY()}>
+  <div class="w-full m-auto">
+    <form on:submit|preventDefault={fetchData}>
+      <div class="flex flex-row flex-wrap items-center justify-center">
+        <GameModeSelect bind:game bind:mode all_option hide_labels is_team />
+        <div class="flex flex-col">
+          <div class="ml-1">
+            <input class="w-44" name="from" type="date" bind:value={from} />
+          </div>
+          <div class="ml-1">
+            <input class="w-44" name="to" type="date" bind:value={to} />
+          </div>
+        </div>
+        <div class="ml-1 my-2">
+          <Button type="submit">Filter</Button>
+        </div>
+      </div>
+    </form>
+    <!-- Solo Tournaments -->
+    <h2 class="text-2xl font-bold">{$LL.TOURNAMENT_HISTORY.SOLO_TOURNAMENTS()}</h2>
+    <div>
+      <Table>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th class="mobile-hide">Date</th>
+            <th class="mobile-hide">Partners</th>
+            <th>Placement</th>
+          </tr>
+        </thead>
+        <tbody>
+          {#each filtered_solo_placements as placement, i}
+            <tr class="row-{i % 2} {placement.placement ? podium_style[placement.placement] : ''}">
+              <td>
+                <a
+                  class="hover:text-emerald-400"
+                  href="/{$page.params.lang}/tournaments/details?id={placement.tournament_id}"
+                >
+                  {placement.tournament_name}
+                </a>
+              </td>
+              <td class="mobile-hide">
+                {toDate(placement.date_start)}
+                {placement.date_end == placement.date_start ? '' : ' - ' + toDate(placement.date_end)}
+              </td>
+              {#if placement.partners != null}
+                <td class="mobile-hide">
+                  {#each placement.partners as partner}
+                    <div class="flex flex-row">
+                      <div class="hover:text-emerald-400">
+                        <PlayerName player_id={partner.player_id} name={partner.player_name} />
+                      </div>
+                    </div>
+                  {/each}
+                </td>
+              {:else}
+                <td></td>
+              {/if}
+              <td>
+                {#if placement.is_disqualified}
+                  Disqualified
+                {:else}
+                  {placement.placement ? placement.placement : '-'}
```

Comment:
We should have ordinal suffixes (1st, 2nd, 3rd) etc for placements, but I looked into it and it seems actually pretty complicated to implement for multiple locales and would take a good amount of time so you don't have to do it for now. If you want you could just write a util function like the top answer here and display them in English for now:
https://stackoverflow.com/questions/13627308/add-st-nd-rd-and-th-ordinal-suffix-to-a-number

Reply:
Done. 👍